src/index.html

```diff
@@ -11,6 +11,43 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="http://localhost:8080/api"
+      method="post"
+      data-qa="big"
+      class="form--element"
+    >
+      <label class="form--element--inner">
```

Comment:
Better use BEM))

https://getbem.com/

```suggestion
      <label class="form__label">
```